docs/index.md

```diff
@@ -81,3 +86,257 @@ They should be written alone on a single line.
+
+![Press +](create-room.png)
+
+Navigate to the "draw tiles" mode (pencil on the bottom-left corner of the screen), select a tile from the panel and start drawing on the canvas on the left!
+
+![Paint the world with sprites](paint-room.png)
+
+Once you are happy with the way the room looks, you need to set up walls so your avatar can't just walk trhough the walls you just drew with tiles.
+
+Go to the "draw walls" mode (briked wall icon on the bottom-left corner of the screen) and select what tiles are not walkable!
+
+![Set up the walls](walls-room.png)
+
+If you want to create rooms quickly, you can also duplicate them! Instead of pressing + in the "switch rooms" panel, press the copy button.
+
+![copy the room to create a duplicate](copy-room.png)
+
+I will use this shortcut to create a mirrored version of this first room, and then tweak the tiles and walls.
+
+Now, you need to set up the room in ink! Navigate to `edit story` (the tab on the right panel with the splotch of ink icon).
+
+There's many ways to go around this, but the simples method is to set up every room as a knot:
+```
+=== left_room
+
+-> left_room
+
+=== right_room
+-
+-> right_room
+```
+
+### Placing the avatar in the new room
+
+Feel free to erase the binksi demo rooms! Open the switch rooms panel and bin them!
+
+However, you have now deleted the player avatar! Do not fret, you can just create it again by going to the "edit events" mode (the bust picture icon under the left canvas). Select an empty space with the reticule and click on the `player avatar` button.
+
+![Use the Player Avatar template](create-avatar.png)
+
+Place the avatar somewhere inside the room.
+
+(Using the Player Avatar button you can respawn the avatar in a new room if you have lost it in one of your rooms and can't find it.)
+
+### Creating an exit
+
+Now we need an exit to go from one room to another! In bipsi you'd use an `exit` event. However, in binksi this works a bit different!
+
+Go to the "edit events" mode, select the space where you want to place the event with the reticule and use the `tag` template.
+
+![Use the Tag template](add-tag.png)
+
+Change the name of the tag from `tagname` to something a bit more descriptive, like `left-room-exit`.
+
+In ink we now create an option in each room so we can interact with the exit tags, and teleport the avatar to another room:
+
+```
+=== left_room
+
++ [tag: left-room-exit]
+    SPAWN_AT(right-room-exit)
+    -> right_room
+
+-
+-> left_room
+
+
+=== right_room
+
++ [tag: right-room-exit]
+    SPAWN_AT(left-room-exit)
+    -> left_room
+-
+-> right_room
+```
+
+As you can see, we are not only using the binksi `SPAWN_AT()` function, but you also need to divert the flow of the ink to the new room. If you forget to do this, the avatar will quite literally be unable to interact with the world, as it'll have lost the flow of the script!
```

Comment:
Maybe add another sentence to point out that at the end of each location-knot you have to loop over itself for the same reason